### PR TITLE
WeTek_Hub: Add libcec/aocec support

### PR DESCRIPTION
**Pull request by Kwiboo**
This updates linux-amlogic-3.14 to LibreELEC/linux-amlogic#3 and adds @gdachs libcec patch for the WeTek Hub.
Replaced the temporarily_use_hdmimode_param_instead_of_vout.patch with a different version as the current patch failed to apply.

Do not merge until LibreELEC/linux-amlogic#3 has been merged and this has been rebased with an updated linux-amlogic-3.14 sha commit.

### CODE REVIEW

File: projects/WeTek_Hub/linux/linux.aarch64.conf [hunk 1]
```diff
@@ -34,7 +39,7 @@ CONFIG_CROSS_COMPILE=""
 CONFIG_LOCALVERSION=""
 # CONFIG_LOCALVERSION_AUTO is not set
 CONFIG_DEFAULT_HOSTNAME="@DISTRONAME@"
-CONFIG_SWAP=y
+# CONFIG_SWAP is not set
```
Comment (codesnake) [inline projects/WeTek_Hub/linux/linux.aarch64.conf]: Why you did that?

Comment (Kwiboo) [inline projects/WeTek_Hub/linux/linux.aarch64.conf]: I synced the config to the generated `.config`, and it will disable swap as `SWAP_SUPPORT="no"` is in `options`

Comment (codesnake) [inline projects/WeTek_Hub/linux/linux.aarch64.conf]: I think it is better to not introduce implicit changes. If you want do disable swap you should do separate commit and explain why toy did that.

Comment (Kwiboo) [inline projects/WeTek_Hub/linux/linux.aarch64.conf]: It is only disguised as enabled in the current `linux.aarch64.conf `-file, at compile time it will be disabled due to the `options`-file. So I guess we should make a PR to enable swap?

Comment (Kwiboo) [inline projects/WeTek_Hub/linux/linux.aarch64.conf]: I see https://github.com/Kwiboo/LibreELEC.tv/blob/c2/packages/linux/package.mk#L105 now, so will change back.

**Comment by codesnake** [comment]
You can check here: https://github.com/codesnake/LibreELEC.tv/commits/le_hosted_aml_kernel. I was about to PR the same changes as you, but you was the fisrt :). I also synced with .config, but CONFIG_SWAP didn't change.

File: projects/WeTek_Hub/linux/linux.aarch64.conf (hunk 1 above)
Comment (codesnake) [inline projects/WeTek_Hub/linux/linux.aarch64.conf]: You can check here: https://github.com/codesnake/LibreELEC.tv/commits/le_hosted_aml_kernel. I was about to PR the same changes as you, but you was the fisrt :). I also synced with .config, but CONFIG_SWAP didn't change.

**Comment by Kwiboo** [comment]
Updated to match @codesnake's `linux.aarch64.conf` with the exception of `CONFIG_IR_MESON=m` and the `CONFIG_AML_AO_CEC` required changes.

File: projects/WeTek_Hub/linux/linux.aarch64.conf (hunk 1 above)
Comment (codesnake) [inline projects/WeTek_Hub/linux/linux.aarch64.conf]: I didn't say CONFIG_SWAP is needed, I said that it shouldn't be changed implicitly.

**Comment by codesnake** [comment]
Thanks @Kwiboo :+1:

**Comment by Kwiboo** [comment]
Rebased and updated to use the merge commit sha of LibreELEC/linux-amlogic#3.
